index.css

```diff
@@ -25,424 +17,1060 @@ a {
+    font-size: 16px;
+    font-weight: 400;
+    margin-left: 27px;
+    margin-right: 27px;
+    margin-top: -10px;
+    margin-bottom: 27px;
+  }
+
+  .about-me button:hover {
+    background-color: white;
+    color: black;
+    cursor: pointer;
+  }
+
+  .about-me hr {
+    margin-left: 27px;
+    margin-right: 27px;
+    margin-bottom: 30px;
+  }
+
+  .lists {
+    display: flex;
+    flex-direction: column;
+    padding-bottom: 20px;
+  }
+
+  .lists li {
+    text-align: center;
+    align-items: center;
+    font-family: 'Inter', sans-serif;
+    list-style: none;
+    text-decoration: none;
+    margin-bottom: 30px;
+  }
+
+  .lists h3 {
+    font-family: 'Inter', sans-serif;
+    font-size: 32px;
+    text-align: center;
+  }
+
+  .attach-icon {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .interesting {
+    width: 375px;
+  }
+
+  .interest-text {
+    font-family: 'Inter', sans-serif;
+    text-align: center;
+    margin-top: 80px;
+    font-size: 32px;
+    font-weight: 700;
+    margin-bottom: 60px;
+    margin-left: 10px auto;
+    margin-right: 50px auto;
+  }
+
+  .interesting input,
+  textarea {
+    font-family: 'Inter', sans-serif;
+    width: 375px;
+  }
+
+  .interesting input {
+    font-size: 15px;
+    border: none;
+    margin-bottom: 2px;
+  }
+
+  .end-name {
+    margin-bottom: 50px;
+    border-color: rgba(219, 216, 215, 0.904);
+    margin-left: 0;
+  }
+
+  .end-address {
+    margin-bottom: 50px;
+    border-color: rgba(219, 216, 215, 0.904);
+    margin-left: 0;
+  }
+
+  .interesting .text-area {
+    font-size: 15px;
+    font-style: italic;
+    background-color: #fbf8f7;
+    border: none;
+    margin-bottom: 5px;
+  }
 
-.lists h3 {
-  font-family: 'Inter', sans-serif;
-  font-size: 32px;
-  text-align: center;
+  .interesting button:hover {
+    background-color: white;
+    color: black;
+    border-color: black;
+  }
+
+  .end-page {
+    border-color: rgba(219, 216, 215, 0.904);
+    margin-bottom: 27px;
+    width: 375px;
+    margin-left: auto;
+  }
+
+  .footer-icons {
+    display: flex;
+    list-style-type: none;
+    align-items: center;
+    justify-content: center;
+    margin-right: 0;
+    margin-bottom: 50px;
+  }
+
+  .footer-icons li {
+    align-items: center;
+    justify-content: center;
+    margin-right: 30px;
+  }
+
+  .footer ul {
+    color: #505f79;
+    margin-left: 80px;
+    margin-right: 0;
+    padding-left: 2px;
+    align-items: left;
+    justify-content: left;
+  }
+
+  .footer hr {
+    margin-left: 125px;
+    margin-right: 30px;
+    border-color: black;
+    border-top: 5px solid;
+    border-radius: 60px 60px 60px 60px;
+    width: 134px;
+    justify-content: center;
+  }
 }
 
-.attach-icon {
-  display: flex;
-  align-items: center;
-  justify-content: center;
+/* ***************************************************************************************************************** */
+@media (min-width: 768px) {
+  main {
```

Comment:
- [ ] As mentioned in the previous CR, your application is not responsive on this screen size YET, I think you should have started the desktop design from this, and the value above it should match or be easily modified, see the record below, 

![6](https://github.com/anyars-encarta/Porfolio/assets/77843050/64ceee69-a988-4f16-b51d-31d16c0c0178)